data/FrancescoXX.json

```diff
@@ -68,6 +68,12 @@
   "tags": [
     "Open Source",
     "Javascript",
+    "Typescript",
+    "JavaScript",
```

Comment:
Hey Francesco, we don't need to use the same tag twice. Please remove the duplicate tag here. 🛠️ 

```suggestion
    "Javascript",
    "Typescript",
```